libraries/apollo-compiler/api/apollo-compiler.api

```diff
@@ -101,6 +101,21 @@ public final class com/apollographql/apollo3/compiler/ExpressionAdapterInitializ
 	public final fun serializer ()Lkotlinx/serialization/KSerializer;
 }
 
+public final class com/apollographql/apollo3/compiler/GeneratedMethod : java/lang/Enum {
+	public static final field COPY Lcom/apollographql/apollo3/compiler/GeneratedMethod;
+	public static final field Companion Lcom/apollographql/apollo3/compiler/GeneratedMethod$Companion;
+	public static final field DATA_CLASS Lcom/apollographql/apollo3/compiler/GeneratedMethod;
+	public static final field EQUALS_HASH_CODE Lcom/apollographql/apollo3/compiler/GeneratedMethod;
+	public static final field TO_STRING Lcom/apollographql/apollo3/compiler/GeneratedMethod;
+	public static fun valueOf (Ljava/lang/String;)Lcom/apollographql/apollo3/compiler/GeneratedMethod;
+	public static fun values ()[Lcom/apollographql/apollo3/compiler/GeneratedMethod;
+}
+
+public final class com/apollographql/apollo3/compiler/GeneratedMethod$Companion {
+	public final fun defaultsFor (Lcom/apollographql/apollo3/compiler/TargetLanguage;)Ljava/util/List;
+	public final fun fromName (Ljava/lang/String;)Lcom/apollographql/apollo3/compiler/GeneratedMethod;
+}
```

Comment:
This should probably be internal too.

Reply:
Unfortunately I think it needs to be public because `DefaultService` lives in a different module

Reply:
Aahh yes, good call. We have this `@ApolloInternal` annotation that we use in such cases. Not perfect but still something.